tools/vmd/ssh_pass.c

```diff
@@ -0,0 +1,509 @@
+/*  This file is part of "sshpass", a tool for batch running password ssh authentication
```

Comment:
This is atleast a record of the canonical source code I use to compile it. Otherwise, you would have to ensure downloading the identical source from the tens of URLS that come up on a google search for this code.